blog/2024-09-04-lawn-mower-activity-returning.md

```diff
@@ -0,0 +1,32 @@
+---
+author: Michael Arthur
+authorURL: https://github.com/mikey0000
+title: "New returning type in LawnMowerActivity"
+---
+
+As of Home Assistant Core 2024.9, integrations implementing `LawnMowerEntity` can indicate that their mower is returning to the dock by using the new `LawnMowerActivity.RETURNING` state.
+
+## Example
+
+```python
+    async def async_dock(self) -> None:
+        """Start docking."""
+        self._attr_activity = LawnMowerActivity.RETURNING
+        self.async_write_ha_state()
+```
+
+or by returning `LawnMowerActivity.RETURNING` in the activity property of your `lawn_mower` entity.
+
+```python
+    @property
+    def activity(self) -> LawnMowerActivity:
+        """Return the state of the mower."""
+        mower = self.mower
+        if mower.state is RETURNING:
+            return LawnMowerActivity.RETURNING
+        ...
+```
+
+## More information
+
+More details can be found in the [documentation](https://developers.home-assistant.io/docs/core/entity/lawn-mower#activities)
```

Comment:
Use relative linking and I think you could move this up to where the comment was made instead of here in the bottom.

Reply:
moved and made relative, probably want to make the previous blog post relative too then.